content/post/data-dies-in-darkness.md

```diff
@@ -0,0 +1,73 @@
+
+Businesses (and especially startups!) can change rapidly, and the data and reporting tools need to be able to change just as fast if they are to remain relevant. Training analysts to build and maintain these data tools is crucial to building a data platform that is sufficiently flexible to support the shifting needs of the business.
+
+#### Identify the directly-responsible-individual for every business KPI
+The easiest way to incentivize someone to monitor data quality is to make a metric part of their performance review. The web product team should be responsible for conversion rates, and the distribution team should be responsible for shipping times. If you can surface the KPIs that are most important to their roles (in partnership with them, as the domain experts) you will have recruited a data quality auditor who jumps out of bed every morning to check on their performance. If something screwy is going on with the data, trust me, you'll hear about it.
+
+#### Automate all data-driven workflows
+In addition to holding teams accountable for goals and metrics, an analytics team can drive data quality partnership by automating manual but data-driven tasks. If you can automate the task for someone (provide them value) while still keeping them responsible for the output, you can get their ongoing help in monitoring the ongoing data quality. The key here is that people are often doing "data quality" monitoring during their workflows, but those checks never get surfaced outside of that individual's workflow. You really want that person to be checking data quality in your central data warehouse so that way everyone in the company benefits.
+
+For example, maybe it's someone's job to once-a-month report out on distribution costs by product mix. Because the systems weren't designed to be integrated, this involves some fairly tedious excel work (download some stuff from system x, find the email from person y, index-match and pivot). In this work-flow, if there's any data issues, they are quickly caught by the person doing this reporting work ("that doesn't look right, I need to check z.") 
+
+If you can automate this workflow, you can automate the data quality checks, and rather than having a person doing quality checks on their local lap-top, you can get them checking the warehouse for quality (where everyone benefits). For example, maybe you can scrape the report from system z, automatically capture excel reports sent via email, and perform the remaining joins and summaries in SQL.
```

Comment:
`lap-top`?  we can all agree it's "laptop", right?

Reply:
lol thank you 🙏